src/lib/UI/Module/ContentTree/NodeFactory.php

```diff
@@ -340,6 +354,7 @@ private function buildNode(
         $children = [];
         if ($loadChildren && $depth < $this->getSetting('tree_max_depth')) {
             $searchResult = $this->findSubitems($location, $limit, $offset, $sortClause, $sortOrder);
+            /** @var int $totalChildrenCount */
```

Comment:
Is this needed? Wouldn't cast to int work in this case?

Reply:
I've added it because it can be `int` or `null`.`totalCount` returns
```
`null` if Query->performCount was set to false and search engine avoids search lookup.
```
and I concluded that in this case we can tell the phpstan that it will be always `int` here.
Should this be changed?

Reply:
I changed it to casting.